src/main/java/com/hivemq/bootstrap/HiveMQNettyBootstrap.java

```diff
@@ -291,14 +291,12 @@ private static class UpdateGivenFutureListener implements ChannelFutureListener
         @Override
         public void operationComplete(final @NotNull ChannelFuture future) throws Exception {
```

Comment:
Noticed "Exception 'java.lang.Exception' is never thrown in the method" in IntelliJ:
```suggestion
        public void operationComplete(final @NotNull ChannelFuture future) {
```